app/invocation/invocation_timing_card.tsx

```diff
@@ -223,11 +227,13 @@ export default class InvocationTimingCardComponent extends React.Component {
           <div className="content">
             <div className="header">
               <div className="title">All events</div>
-              <div className="button">
-                <Button className="download-gz-file" onClick={this.downloadProfile.bind(this)}>
-                  Download profile
-                </Button>
-              </div>
+              {this.state.timingEnabled && !this.state.isMissingProfile && this.state.timingLoaded && (
```

Comment:
I think this could just be `this.state.timingLoaded`, since the other cases would have to be false if `this.state.timingLoaded` is true.